Gitpod-Samples/spring-petclinic
# Use the auto-generated .idea directory

## Description

Since we deployed https://github.com/gitpod-io/gitpod/pull/14656, we don't need the `.idea/spring-petclinic.iml` and `.idea/modules.xml` files anymore. But we still need the `.idea/misc.xml` for the warmup to work.

Also, with these other files in the `.idea` folder (generated automatically), the IDE takes the lowest possible time to load, as everything is pre-configured.

## How to test

You can test it by opening this prebuilt branch in IntelliJ IDEA: https://gitpod.io/#referrer:jetbrains-gateway:intellij/https://github.com/Gitpod-Samples/spring-petclinic/tree/use-auto-generated-idea-directory

Confirm it loads as fast as the main branch: https://gitpod.io/#referrer:jetbrains-gateway:intellij/https://github.com/Gitpod-Samples/spring-petclinic
